examples/gatsby/src/pages/cache-healthcheck.txt

```diff
@@ -0,0 +1,5 @@
+import * as React from "react"
```

Comment:
@chriszarate - Only one question here. I'm curious, the file extension `.txt` is to prevent the issue reported on README 2.1 item?

Reply:
+1 -- this file might be worth discussing in the README

Reply:
Good catch @luiztiago. This was an accidental commit of a file that is no longer needed. Removed in https://github.com/Automattic/vip-go-node-skeleton/pull/29/commits/c225286df896cce0d12c642ce00580f1497377b5